P/PCRE2/Compat.toml

```diff
@@ -6,3 +6,6 @@ BinaryProvider = "0.1-0.5"
 
 ["0.2"]
 julia = "1.0-1.3"
+
+["1.0"]
+julia = "1.0-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.